pkg/management/endpoint.go

```diff
@@ -21,6 +21,8 @@ type Endpoints struct {
 	UpdateUser                     endpoint.Endpoint
 	GetUsers                       endpoint.Endpoint
 	CreateUser                     endpoint.Endpoint
+	GetRolesOfUser                 endpoint.Endpoint
+	GetGroupsOfUser                endpoint.Endpoint
 	GetUserAccountStatus           endpoint.Endpoint
 	GetClientRoleForUser           endpoint.Endpoint
 	AddClientRoleToUser            endpoint.Endpoint
```

Comment:
GetRealmRoleForUser endpoint not removed ?

Reply:
Thanks, fixed